spec/services/expense_sheet_generator_spec.rb

```diff
@@ -20,7 +20,7 @@
       end
     end
 
-    it 'sets correct civil_servant', :aggregate_failures do
+    it 'sets correct civil_servants', :aggregate_failures do
```

Comment:
Aggregation is now default, you can remove it

```suggestion
    it 'sets correct civil_servants' do
```